docs/data-collection/scanning-notes.md

```diff
@@ -36,6 +36,11 @@
 
     ![adjustFOV.jpg](../assets/images/adjustFOV.jpg)
 
+### Interrupted scan
+- [ ] In case, you have to interrupt a sequence because a problem occured (e.g your participant fell asleep, the physiological recordings did not start etc.), click right on the sequence that needs to be restarted and click on `Repeat` to restart the scan **without changing anything in its name!**
```

Comment:
```suggestion
- [ ] In case, you have to interrupt a sequence because a problem occured (e.g your participant fell asleep, the physiological recordings did not start etc.), right click on the sequence that needs to be restarted and click on `Repeat` to restart the scan **without changing anything in its name!**
```